sonar-plugin-api/src/main/java/org/sonar/api/batch/Initializer.java

```diff
@@ -28,9 +28,11 @@
  * Initializer are executed at the very beginning of each module analysis, prior the core listing files to be analyzed. It means {@link FileSystem} should not be accessed.
  * <p>
  * @since 2.6
+ * @deprecated since 6.6 no known valid use case
  */
 @ScannerSide
 @ExtensionPoint
+@Deprecated
 public abstract class Initializer implements CheckProject {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)